wled00/wled14_colors.ino

```diff
@@ -17,6 +17,29 @@ void colorFromUint32(uint32_t in, bool secondary)
   }
 }
 
+//load a color without affecting the white channel
+void colorFromUint24(uint32_t in, bool secondary = false)
+{
+  if (secondary) {
+    colSec[0] = in >> 16 & 0xFF;
+    colSec[1] = in >> 8  & 0xFF;
+    colSec[2] = in       & 0xFF;
+  } else {
+    col[0] = in >> 16 & 0xFF;
+    col[1] = in >> 8  & 0xFF;
+    col[2] = in       & 0xFF;
+  }
+}
+
+//relatively change white brightness, minumum A=5
+void relativeChangeWhite(int8_t amount, byte lowerBoundary =0)
+{
+  int16_t new_val = (int16_t) col[3] + amount;
+  if (new_val > 0xFF) new_val = 0xFF;
+  else if (new_val < lowerBoundary) new_val = lowerBoundary;
+  col[3] = new_val;
+}
+
```

Comment:
Change the white channel without any other RGB value being influenced  ---  function is used in WLED20_ir.ino.